### PR TITLE
Remove sysbox-mgr data-root dir when sysbox-mgr is stopped.

### PR DESCRIPTION
**Pull request by ctalledo**
Signed-off-by: Cesar Talledo <ctalledo@nestybox.com>

### CODE REVIEW

File: utils.go [hunk 1]
```diff
@@ -442,31 +442,14 @@ func cleanupWorkDirs() error {
...
 	}
 
-	// Remove the sysbox lib dir's content.
-	if _, err := os.Stat(sysboxLibDir); err == nil {
-		if err := removeDirContents(sysboxLibDir); err != nil {
-			return err
-		}
-	} else if !os.IsNotExist(err) {
-		return err
+	// Remove the sysbox lib dir
```
Comment (rodnymolina) [inline utils.go:445]: Have you checked this works in our test-setup? sysboxLibDir is a bind-mount within the priv container so don't think you can delete the folder itself ...

**Review by rodnymolina — COMMENTED** [review]
See comment below.

File: utils.go (hunk 1 above)
Comment (ctalledo) [reply utils.go:445]: Correct ... I had to make a change in the test container for this to work. PR coming soon.

**Review by rodnymolina — APPROVED** [review]
A new PR will be submitted to correct the issue mentioned above.

**Pull request merged**